docs/mobile-apps/faq.md

```diff
@@ -61,6 +61,16 @@ Yes. See [Camera Image Injection](/mobile-apps/features/camera-image-injection).
 For real devices on the public cloud, the limit is 6 hours for Appium tests and 1 hour for Espresso and XCUITest; the same applies to private cloud devices.
 The limits for public cloud devices ensure that there are enough devices available for you, whereas the limits for private cloud devices ensure the longevity of your devices.
 
+#### **Is there a difference in devices I can use as a free trial and with a subscription when live testing?**
```

Comment:
```suggestion
#### **What is the difference between the devices available with a free trail and a subscription?**
```

Reply:
Please double check typo on trial - here it is trail

Reply:
fixed, thanks.